### DISCUSSION TITLE
Minimize and maximize

### DISCUSSION THREAD

**Discussion by aud-jeremy** [zauberzeug/nicegui#2665] [Q&A] [answered]
### Question

Hello,

I'm looking for an auto minimize and maximize my windows (in native mode or not).

I have found this but it's with tkinter 
minimize : tkWindow.state(newstate='iconic')
maximize : tkWindow.state(newstate='normal')

I dind'nt found the same for nicegui.

Have a good day.

**Comment by falkoschindler** [comment, marked as answer]
Hi @aud-jeremy,

In native mode you can access pywebview's window methods like this:
```py
ui.button('Toggle fullscreen', on_click=lambda: app.native.main_window.toggle_fullscreen())
ui.button('Minimize', on_click=lambda: app.native.main_window.minimize())
```

**Reply by aud-jeremy** [reply]
Ok thanks you.

I try this asap.

It's work. I change toggle_fullscreen() by restore (my app not use fullscreen)
